.github/ISSUE_TEMPLATE/3-feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Use this issue for suggesting new features
+---
+
+<!--
+
+Welcome!  If you have a feature you like us to implement you can suggest here
+-->
+
+### Feature request
+
+
+### What is motivation or use case for adding/changing the behavior?
+
+
+### How should this be implemented in your opinion?
+
+
+### Would you be willing to work on adding this support? 
```

Comment:
I'm not sure about this question, as it may mean different things to different people. I suspect we'd get a lot of "see my already completed PR #0000" and a lot of "Oh, maybe once I've finished my one to one scale model of earth"
I think different sections of open source have different protocols for "claiming" a bug report. I don't know what ours should be. I tend to assume that if there's no open PR, it's unclaimed, but I honestly don't know.

Reply:
I agree, I think if the requester is interested in implementing it themselves, more often than not they make it clear or open a PR for it. If someone answers "yes" to this question, and then changes their mind, I think we'll start getting a lot of "@whoever are you still working on this" type questions on feature request PRs. I don't think the question adds much anyway.

Reply:
Yeah, personally, I don't think anyone would ever want to answer "no" to this, so I think it would be just fine to leave the question off.